### ISSUE TITLE
conda install -c conda-forge cc-plugin-ugrid didn't work 

### ISSUE THREAD

**Issue by drf5n** [ioos/cc-plugin-ugrid#4]
I tried the suggested install method, and it did not work for me:

```
(python3) 649 [drf@V21837 tmp]$ conda install -c conda-forge cc-plugin-ugrid
Fetching package metadata ...............

PackageNotFoundError: Packages missing in current channels:
            
  - cc-plugin-ugrid

We have searched for the packages in the following channels:
            
  - https://conda.anaconda.org/conda-forge/osx-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/sean/osx-64
  - https://conda.anaconda.org/sean/noarch
  - https://repo.continuum.io/pkgs/main/osx-64
  - https://repo.continuum.io/pkgs/main/noarch
  - https://repo.continuum.io/pkgs/free/osx-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/osx-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/osx-64
  - https://repo.continuum.io/pkgs/pro/noarch
            

(python3) 650 [drf@V21837 tmp]$ 

```

**Comment by Bobfrat** [comment]
Good catch, this is a new repo and I don't think we have published to conda yet. You'll have to use pip for now. I'll update the README to reflect this until it gets up.

**Comment by ocefpaf** [comment]
Working on it. I had no idea this was ready for use and had a release on PyPI.

**Comment by ocefpaf** [comment]
> You'll have to use pip for now.

Actually I could not find a PyPI release. Are you referring to pip install from the source here?
Feel free to ping me if you make an official release and I'll package it to in conda-forge,

**Comment by Bobfrat** [comment]
@ocefpaf you're right there hasn't been an official release yet for this (so its not on PyPI either).

**Comment by drf5n** [comment]
So if conda and PyPi aren't yet ready in the current pre-official release state, maybe the current README Install directions should note things that work like:

    git clone https://github.com/ioos/cc-plugin-ugrid.git
    cd cc-plugin-ugrid
    python setup.py install

or
 
     pip install git+https://github.com/ioos/cc-plugin-ugrid.git

I also used a line like:

    touch /Users/drf/anaconda/envs/python3/lib/python3.5/site-packages/VERSION

to get past an error like:

```
(python3) 657 [drf@V21837 tmp]$ compliance-checker -l
Could not load ugrid-1.0 = cc_plugin_ugrid.checker_10:UgridChecker10 : [Errno 2] No such file or directory: '/Users/drf/anaconda/envs/python3/lib/python3.5/site-packages/VERSION'
IOOS compliance checker available checker suites (code version):
 - acdd (3.0.4)
 - acdd:1.1 (3.0.4)
 - acdd:1.3 (3.0.4)
 - acdd:latest (3.0.4)
 - cf (3.0.4)
 - cf:1.6 (3.0.4)
 - cf:latest (3.0.4)
 - ioos (3.0.4)
 - ioos:0.1 (3.0.4)
 - ioos:latest (3.0.4)
(python3) 658 [drf@V21837 tmp]$ 

```

**Comment by ocefpaf** [comment]
It has been a while that this plugin in available on `conda-forge`, even the latest v1.0 (PyPI still has only v0.1.0 though). Closing this.
